src/FixerDefinition/FixerDefinition.php

```diff
@@ -36,9 +36,9 @@ public function __construct(
         $summary,
         $description,
         array $codeSamples,
-        $configurationDescription,
+        $configurationDescription = null,
         array $defaultConfiguration = null,
-        $riskyDescription
+        $riskyDescription = null
```

Comment:
this is not static code analysis (like phpmd) problem. it's missing argument in your code.